core/tests/puppeteer-acceptance-tests/utilities/user/topic-manager.ts

```diff
@@ -257,14 +259,127 @@ const removeSkillModalHeaderSelector =
 const addMisconceptionHeaderSelector =
   '.e2e-test-oppia-misconception-card-header';
 
+const topicAndSkillsOptionInProfileMenu =
+  '.e2e-test-topics-and-skills-dashboard-link';
+const topicAndSkillsDashboardPageSelector =
+  '.e2e-test-topic-and-skills-dashboard';
+const navbarBreadcrumbSelector = 'oppia-navbar-breadcrumb';
+const resetTopicFilterButtonSelector = '.e2e-test-topic-filter-reset';
+
+const floatTextField = '.e2e-test-rule-details .e2e-test-float-form-input';
+const solutionFloatTextField =
+  'oppia-add-or-update-solution-modal .e2e-test-float-form-input';
+const textStateEditSelector = 'div.e2e-test-state-edit-content';
+const saveContentButton = 'button.e2e-test-save-state-content';
+const createQuestionButton = 'div.e2e-test-create-question';
+const addInteractionButton = 'button.e2e-test-open-add-interaction-modal';
+const interactionNumberInputButton =
+  'div.e2e-test-interaction-tile-NumericInput';
+const interactionNameDiv = 'div.oppia-interaction-tile-name';
+const saveInteractionButton = 'button.e2e-test-save-interaction';
+const responseRuleDropdown =
+  'oppia-rule-type-selector.e2e-test-answer-description';
+const equalsRuleButtonText = 'is equal to ... ';
+const answersInGroupAreCorrectToggle =
+  'input.e2e-test-editor-correctness-toggle';
+const saveResponseButton = 'button.e2e-test-add-new-response';
+const defaultFeedbackTab = 'a.e2e-test-default-response-tab';
+const openOutcomeFeedBackEditor = 'div.e2e-test-open-outcome-feedback-editor';
+const saveOutcomeFeedbackButton = 'button.e2e-test-save-outcome-feedback';
+const openAnswerGroupFeedBackEditor = 'i.e2e-test-open-feedback-editor';
+const addHintButton = 'button.e2e-test-oppia-add-hint-button';
+const saveHintButton = 'button.e2e-test-save-hint';
+const addSolutionButton = 'button.e2e-test-oppia-add-solution-button';
+const answerTypeDropdown = 'select.e2e-test-answer-is-exclusive-select';
+const submitAnswerButton = 'button.e2e-test-submit-answer-button';
+const submitSolutionButton = 'button.e2e-test-submit-solution-button';
+const saveQuestionButton = 'button.e2e-test-save-question-button';
+
+// Topic Editor > Preview Tab.
```

Comment:
This comment isn't super clear. How about "Preview tab of the topic editor" or similar instead?

ditto elsewhere

Reply:
Done.